components/shared/is-wallet-disconnected.tsx

```diff
@@ -0,0 +1,17 @@
+'use client'
+
+import { ReactNode } from 'react'
+
+import { useAccount } from 'wagmi'
+
+interface IsWalletDisconnectedProps {
```

Comment:
Add generic React Typescript props so don't need to redeclare children.

Reply:
Actually in this case the only possible prop is a required `children` with no extra props from any html element.